content/ja/handbook/customer-success/professional-services-engineering/project-mgmt/index.md

```
@@ -0,0 +1,216 @@
+---
+title: "プロフェッショナルサービス プロジェクトマネジメント"
+description: "GitLab PMO & Delivery チームは、プロフェッショナルサービスのデリバリーにおける卓越性を通じて、お客様の Time to Value を加速します。[PS Customer Journey](https://lucid.app/lucidchart/13d16253-7bba-4082-9b72-761c6105c7ed/edit?viewport_loc=-24156%2C-2407%2C3862%2C1831%2Cm-5o7ONTd-nK&invitationId=inv_6a3429e9-eb45-4a96-b300-dc11bd10ff35) を通じて、合意されたスコープ・タイムライン・予算内でプロジェクトを提供し、変化する顧客ニーズに対応するイテレーションを取り入れています。私たちの成功は、お客様が GitLab 製品の利点を完全に享受できるソリューションを提供することで測られます。PS の提供メニューは [こちら](https://about.gitlab.com/services/) でご確認いただけます。"
+upstream_path: /handbook/customer-success/professional-services-engineering/project-mgmt/
+upstream_sha: b4eeb07f0d5f46e2fc5f8572be1a2547261aed89
+translated_at: "2026-04-26T03:00:00Z"
+translator: claude
+stale: false
+---
+
+- [プロジェクトの開始 & 計画](#project-initiate--plan)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)